pkg/kubelet/apis/deviceplugin/v1beta1/api.proto

```diff
@@ -111,6 +111,7 @@ message AllocateRequest {
 
 message ContainerAllocateRequest {
 	repeated string devicesIDs = 1;
+	map<string, string> podAnnotations = 2;
```

Comment:
Could you please add a comment like: pod annotations are for experiment purpose and do not abuse?

Reply:
ok, I'll add it. Thank you for the review!